source/incidents-and-alerts/team-dashboard.html.md.erb

```diff
@@ -18,20 +18,20 @@ The top monitor has technical dashboards worth keeping an eye on throughout the
 Grafana is an open source analytics and monitoring platform, and we use it for monitoring the health
 of GovWifi in realtime.
 
-GovWifi has its own organisation, and we have admin-level access so we can now make our own changes
-without having to always involve RE.
+We have access to two types of Grafana metrics.
 
-We currently have two dashboards in Grafana, which are both shown on the TV monitor:
+#### Self Hosted
 
-1. The first is for monitoring performance of GovWifi Product Pages and Tech Docs, and is available
-[here](https://grafana-paas.cloudapps.digital/d/KMxSG3DWk/govwifi).
+This is used for monitoring data related to GovWifi's SLIs for authentication journeys, and other Radius metrics. In-depth information on the technical setup of this is available [here](https://docs.google.com/document/d/1mmYZPagv0pPrs4f4iGANEoj5GTMisZU1RyN1qaP5dck/) (you must be a member of the Govwifi team to view this document). You can access this dashboard at the links below (you must be signed into the dashboard to do this):
```

Comment:
It's probably worth adding the self-hosted Grafana is also for general monitoring of GovWifi infrastructure and APIs. I'm thinking about what would be useful for a new joiner to know on day 1.

Grafana should become the single source of truth for monitoring once the dashboards are set up.